mono/btls/MonoBtls.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,393 @@
+// !$*UTF8*$!
```

Comment:
Yeah sure - there's actually a bunch of stuff which can go away.  I simply copied everything because I want to get Jenkins to build this ASAP and I can do the cleanups while waiting for those builds.